### PR TITLE
Make cipher suites configurable via the command line

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

File: src/riak_core_security.erl [hunk 1]
```diff
@@ -407,6 +435,39 @@ is_enabled() ->
...
+        _ ->
+            io:format("Unknown/Unsupported ciphers(~b)~n~n~s~n~n",
+                      [length(Bad), string:join(Bad, ":")])
+    end.
+
+set_ciphers(CipherList) ->
+    case riak_core_ssl_util:parse_ciphers(CipherList) of
+        {[], _} ->
+            %% no valid ciphers
+            io:format("No known or supported ciphers in list."),
```
Comment (bookshelfdave) [inline src/riak_core_security.erl]: might be nice to have a newline at the end of this msg

**Comment by bookshelfdave** [comment]
+1

output:

```
R16B02:[adt-cipher-suites]prime:~/basho/riak$ ./dev/dev1/bin/riak-admin security ciphers ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-Sasdasd
Configured ciphers

ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-Sasdasd

Valid ciphers(1)

ECDHE-RSA-AES128-SHA256

Unknown/Unsupported ciphers(1)

ECDHE-ECDSA-AES128-Sasdasd
```

**Comment by bookshelfdave** [comment]
please rebase against develop and push, riak_core_connection has been removed since and causes build problems.

**Comment by Vagabond** [comment]
Rebased.

**Comment by bookshelfdave** [comment]
retested w/ https://gist.github.com/metadave/8059006

**Comment by bookshelfdave** [comment]
+1

**Pull request merged**